### ISSUE TITLE
descendants - Nested Eager Loading doesn't work?

### ISSUE THREAD

**Issue by FilipQL** [lazychaser/laravel-nestedset#252]
As stated in the documentation, `descendants` can be eagerly loaded:

> Node has many `descendants`

But is it possible to use Nested Eager Loading with `descendants`? 

![eloquent_ relationships - laravel - the php framework for web artisans - mozilla firefox 2017-11-10 01 04 31](https://user-images.githubusercontent.com/18264428/32636029-40a41f3a-c5b3-11e7-8481-89392b4eb76f.png)

For example, something like:

```
        $article->comments()
            ->whereIsRoot()
            ->with('descendants.author')
            ->get();
```

**Comment by nivv** [comment]
@FilipQL have you tried:

```php
        $article->comments()
            ->whereIsRoot()
            ->with('children.author')
            ->get();
```

**Comment by lazychaser** [comment]
Fixed in v4.3.1
